server/api/section/section.model.js

```diff
@@ -17,22 +17,6 @@ var SectionSchema = new Schema({
  * Methods
  */
 SectionSchema.methods = {
-  getFullEvents(cb){
```

Comment:
This is still in server/api/section/section.controller.js line 96
should be removed there as well.